marquez_client/utils.py

```diff
@@ -92,3 +94,8 @@ def now_ms():
     def utc_now():
         return str(generate(datetime.utcnow().replace(tzinfo=pytz.utc),
                             microseconds=True))
+
+    @staticmethod
+    def log_details(details):
+        log_details = json.dumps(details)
+        log.info(log_details)
```

Comment:
Did you intend to have the FileBackend also call this method for logging? One caveat now it's calling the log for the utils instead of the original class log.
I would recommend keeping log.info(json.dumps(details)) in log_backend (and file_backend)

Reply:
i don't mind changing it.
FYI, log.format abstracts that info. so you don't know if it is LogBackend or Utils log.
sample:

`
2020-10-12 14:08:26 DEBUG MarquezWriteOnlyClient.setup(): 
2020-10-12 14:08:26 DEBUG LogBackend.init
2020-10-12 14:08:26 INFO created marquez_client_wo_log.
2020-10-12 14:08:26 INFO test_create_namespace()
2020-10-12 14:08:26 DEBUG _put()
2020-10-12 14:08:26 INFO {"method": "PUT", "path": "/api/v1/namespaces/my-namespace", "headers": {"User-Agent": "marquez-python/0.7.6"}, "payload": {"ownerName": "me", "description": "my namespace for testing."}}

`

Reply:
other backend changes, i will create a new PR.

Reply:
as you prefer. I think it's just a couple of lines.

Reply:
https://docs.python.org/3/library/logging.html#logrecord-attributes
This is how your formatter is configured, you could add %(filename)s and %{lineno)s to your tests/pytest.ini

Reply:
> Did you intend to have the FileBackend also call this method for logging? One caveat now it's calling the log for the utils instead of the original class log.
> I would recommend keeping log.info(json.dumps(details)) in log_backend (and file_backend)

added test_log_backend.py

Reply:
Makes reading message hard.
Makes log file fat.
```
2020-10-12 15:24:03 DEBUG /Users/svaddi/SreeVaddi/source/github/MarquezProject/marquez-python/tests/test_log_backend.py 25 TestLogBackend.setup(): 
2020-10-12 15:24:03 DEBUG /Users/svaddi/SreeVaddi/source/github/MarquezProject/marquez-python/marquez_client/log_backend.py 23 LogBackend.init
2020-10-12 15:24:03 DEBUG /Users/svaddi/SreeVaddi/source/github/MarquezProject/marquez-python/tests/test_log_backend.py 47 TestLogBackend.test_post(): 
2020-10-12 15:24:03 DEBUG /Users/svaddi/SreeVaddi/source/github/MarquezProject/marquez-python/marquez_client/log_backend.py 38 _post()


2020-10-12 15:24:34 DEBUG test_log_backend.py 25 TestLogBackend.setup(): 
2020-10-12 15:24:34 DEBUG log_backend.py 23 LogBackend.init
2020-10-12 15:24:34 DEBUG test_log_backend.py 47 TestLogBackend.test_post(): 
2020-10-12 15:24:34 DEBUG log_backend.py 38 _post()
```

Reply:
If you don't mind, can you change the log call to be directly `log.info(json.dumps(details))` in `log_backend` and `file_backend` ?

Reply:
done.